Sources/Misc/Concurrency/OperationDispatcher.swift

```diff
@@ -108,6 +109,8 @@ private extension Delay {
         case .none: return 0
         case .`default`: return 0
         case .long: return Self.maxJitter
+        case .timeInterval(let timeInterval):
+            return max(timeInterval - 0.1, 0)
```

Comment:
Including a +/- 0.1 here to allow for some jittering

Reply:
we should turn that into a comment

Reply:
do we need it though? the 429's shouldn't be triggered by something like a push notification, so presumably we wouldn't be sending out these requests en masse

Reply:
Fair enough, I'll remove the jittering here!